data/assets/examples/translation/statictranslation/static_timeframe.asset

```diff
@@ -0,0 +1,35 @@
+-- Time Frame
+-- This asset creates a set of coordinate axes that are offset from their original
```

Comment:
This example is a good example of why I think the first sentence should explain the entire example, briefly :) 

All sentences start with either "this asset" or "this example", which to me suggested that the flow of the explanation is wrong